layouts/index.html

```diff
@@ -4,27 +4,7 @@
 {{ partial "article.html" . }}
 {{ end }}
 
-<a href="./news/">» See all articles</a>
-
-<script>
```

Comment:
I had actually put the ical script inline on purpose, to avoid a network round-trip, for case where a human is visiting, which is latency-sensitive and the most common case.

Reply:
Understood - will revert.